src/main/java/mekanism/common/block/BlockTransmitter.java

```diff
@@ -231,49 +230,84 @@ public boolean onBlockActivated(World world, BlockPos pos, IBlockState state, En
 
 		return false;
 	}
-	
-	@Override
-    public ItemStack getPickBlock(IBlockState state, RayTraceResult target, World world, BlockPos pos, EntityPlayer player)
-	{
-		ItemStack itemStack = ItemStack.EMPTY;
+
+	private ItemStack getDropItem(IBlockAccess world, BlockPos pos) {
 		TileEntitySidedPipe tileEntity = getTileEntitySidedPipe(world, pos);
 		if(tileEntity != null)
 		{
-			itemStack = new ItemStack(MekanismBlocks.Transmitter, 1, tileEntity.getTransmitterType().ordinal());
-			
-			if(!itemStack.hasTagCompound())
+			ItemStack itemStack = new ItemStack(MekanismBlocks.Transmitter, 1,
+					tileEntity.getTransmitterType().ordinal());
```

Comment:
this line does not need to be split

Reply:
Removed most IDE and non-essential changes.